source/features/user-local-time.tsx

```diff
@@ -102,6 +103,14 @@ function init(): void {
 			return;
 		}
 
+		const timeout = delay(300);
+		// This will delay adding the icon for 300ms, enough to check if the date is store as "not found"
+		const race = await Promise.race([timeout, getLastCommitDate(login)]);
```

Comment:
`getLastCommitDate(login)` is repeated below. I suggest this or `datePromise`

```suggestion
		const date = getLastCommitDate(login);
		const race = await Promise.race([timeout, date]);
```

Reply:
```suggestion
		const datePromise = getLastCommitDate(login);
		const race = await Promise.race([timeout, datePromise]);
```